docker/bin/await-service

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# Wait for a service to accept connections.
```

Comment:
I like the idea behind this script. Keep in mind that it can lie about etcd. Etcd will open a socket listener before it's actually ready to process API requests. In the DCOS image I've build, I ended up waiting on an HTTP 200 from some service URL instead of host:port.